app/poc-common/form-display/directives/formField.js

```diff
@@ -35,11 +35,21 @@ angular.module('poc.common.formdisplay')
                     if(last.value.datatype && last.value.datatype.display === "Coded") {
                         $scope.fieldModel.value = realValueOfField($scope.fieldModel.fieldConcept.concept.answers, last.value);
                         return;
+                        console.log($scope.fieldModel.value);
```

Comment:
Debugging code like this should be cleaned up.

![](http://blogs.versionone.com/agile_management/wp-content/uploads/sites/3/2013/01/feature.jpg)